test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -398,7 +398,7 @@ private static void addInitWithArgs(ClassWriter cw, String className, String mak
 		mv.visitCode();
 		for (int i = 0, count = 1; i < fields.length; i++) {
 			mv.visitVarInsn(ALOAD, 0);
-			String nameAndSigValue[] = fields[i].split(":");
+			String[] nameAndSigValue = fields[i].split(":");
```

Comment:
While you're here, please correct the indentation of the comments on lines 363-367 and lines 378-381.